scheme/arm-cca/evidence_handler.go

```diff
@@ -10,8 +10,7 @@ import (
 	"fmt"
 
 	"github.com/veraison/ccatoken"
-	"github.com/veraison/ear"
-	"github.com/veraison/psatoken"
+	ar "github.com/veraison/ear"
```

Comment:
Do not alias packge unless absolutely necessary. This makes it harder to read code. When I'm reading a function, I know what an `ear.AttestationResult` is, I won't know whan an `ar.AttestationResult`, and will have to just to the imports to resolve the alias.

Reply:
Thank you for your comment, This specific one on `ear` is addressed once I noticed a similar comment on the other PR. It is now used only when it is `absolutely` necessary! Resolving this comment, as it is addressed!